azure-pipelines.yml

```diff
@@ -494,7 +494,6 @@ jobs:
           CMAKE_OPT+="
             -DWITH_MECAB=system
             -DWITH_NUMA=ON
-            -DWITH_READLINE=system
```

Comment:
May be also add `-DWITH_EDITLINE=system` to `INVERTED`?

Reply:
Done but I decided that inverted will be `bundled` and normal is `system`.